src/index.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
You can also use styled-components for global styles

https://styled-components.com/docs/api#createglobalstyle